Assets/Scripts/Movement.cs

```diff
@@ -30,49 +32,64 @@ public class Movement : MonoBehaviour
     public Transform playerBody;
     private float X_Rotation = 0;
 
-    //For Torch Tossing\\
-    public GameObject Torch;
+    // For Torch Tossing \\
     private InputAction TossInput;
+    public GameObject Torch;
+    public GameObject TorchSpawnPoint;
+    public float UpwardsForce = 5.5f, ForwardsForce = 8f;
```

Comment:
Thanks for putting these in a section with other torch variables, but I think it would be more clear for other parts of the script to add "torch" to the names somehow
```suggestion
    public float torchUpwardsForce = 5.5f, torchForwardsForce = 8f;
```